api/src/dto/ContentDto.ts

```diff
@@ -51,10 +53,11 @@ export class ContentDto extends _contentBaseDto {
     video?: Uuid;
 
     @IsDate()
-    @IsNotEmpty()
+    @IsOptional()
```

Comment:
I think we can require it to be set before saving but by default set it to the document creation date (in the CMS)? The editor should be able to set it to a different date if he wants to.

Will we only include the document in Sync after the publish date? (perhaps not in MVP but later) to enable "scheduled publish".

Reply:
My thinking was that the document might not always be published the same day it was created, and that we can instead provide a dropdown in the CMS with options:
- Set to current date when published
- Custom date.. -> datepicker

So similar to how we said we would handle expiry date.

Then it's also handled for translations created after the original translation was published, and we avoid the problem of the current site where it defaults to a wrong (older) date.